### ISSUE TITLE
NTR: Plant thorn skin puncture (Triggers or responses: non-drug)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9551]
New term request Plant thorn skin puncture (segal_211213192358):   UMLS: 1274587

**Comment by pnrobinson** [comment]
@MickeySegal
SInce "Plant thorn skin puncture" itself can (almost) be understood as a phenotypic feature, could we provide more information about what is being triggered, e.g., a larger skin rash?

**Comment by MickeySegal** [comment]
The plan was for this to be a general historical finding, but we have used it only for "Plant thorn synovitis".

**Comment by pnrobinson** [comment]
@MickeySegal I have created a new term
Recent past medical history
def: History of events that precede the current presenting condition and are thought to be of potential relevance, having occured in the hours, days, or weeks before the current condition.
![image](https://user-images.githubusercontent.com/4943420/213913457-a53430ff-c923-4489-a421-0c7d1ef37075.png)

Should we add this term (and some similar requests) as
Recent plant thorn skin puncture
def: A history of a plant skin puncture preceding the current presenting condition by hours, days, or weeks.

**Comment by MickeySegal** [comment]
That sounds great.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000260
- label: History of recent plant thorn skin puncture
- parents: [Recent past medical history (HP:4000144)](https://hpo.jax.org/app/browse/term/HP:4000144)
- pmid: 
- def: A history of a plant skin puncture preceding the current presenting condition by hours, days, or weeks.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9551
- synonyms:
